extensions/amp-auto-ads/0.1/adsense-network-config.js

```diff
@@ -48,6 +50,16 @@ export class AdSenseNetworkConfig {
     );
   }
 
+  /** @override */
+  filterConfig(config) {
+    // Force no fill on anchor ads if adsbygoogle tag (non-AMP Adsense) is already loaded.
+    // We do this to prevent multiple anchor ads from being loaded.
+    if (getWin(this.autoAmpAdsElement_).adsbygoogle) {
+      config[Attributes.STICKY_AD_ATTRIBUTES]['data-no-fill'] = 'true';
```

Comment:
Do you need to check that `config[Attributes.STICKY_AD_ATTRIBUTES]` exists first?

Reply:
Thanks. I thought we always have that. Updated.